src/org/labkey/test/pages/reports/ManageViewsPage.java

```diff
@@ -63,4 +63,12 @@ public void deleteReport(String reportName)
                 "Failed to delete report: " + reportName, WAIT_FOR_JAVASCRIPT);
     }
 
+    @LogMethod
+    public void editReport(String reportName)
+    {
+        //td[1] is the pencil
+        final Locator report = Locator.tag("tr").withClass("x4-grid-row").containing(reportName).childTag("td").position(1);
+        click(report);
```

Comment:
If this navigates, we should wait for the page load.
```suggestion
        clickAndWait(report);
```

Reply:
It does not navigate, thats only click.